### ISSUE TITLE
CMake error about src/googletest-build/googlemock/gtest/libgtest.a

### ISSUE THREAD

**Issue by fjgmoya** [nmerrill67/GPU_GSPCA#1]
Hi:

When trying to build code (make), I found an error saying "No rules to build objective src/googletest-build/googlemock/gtest/libgtest.a needed for kernel_pca_test. Halt."

Make process finish at this point. Could you help me?
Thanks in advance.

**Comment by nmerrill67** [comment]
Hi thanks for the interest! It looks like the google test repo might be broken! I am seeing the same error. You can skip the unit tests in compilation by doing `cmake -DBUILD_TESTS=0 ..` and it compiles fine for me, but you just can't run the unit tests. I will look into perhaps linking to a working gtest version, but for now this should suffice to get the library built.

**Comment by fjgmoya** [comment]
Thanks for your answer, Nate. 
Executed cmake -DBUILD_TESTS=0 .. with no problems.
When I execute the "make tests" command, system says "there is no rule to construct the objective 'test'. Halt."
I cannot find python pca_kernel library, I suppose it's not created.
Could you help me ?
Thanks in advance.

**Comment by nmerrill67** [comment]
Yes that is because that flag BUILD_TESTS=0 ommits building the testing binaries. You can only link to the libraries and run the main executable.
